codebase/superdarn/src.idl/lib/main.1.25/grd.pro

```diff
@@ -198,13 +199,13 @@ function GridRead,unit,prm,stvec,gvec
            'program.id','noise.mean','noise.sd','gsct', $
            'v.min','v.max','p.min','p.max','w.min','w.max','ve.min', $
            've.max', $
-           'vector.mlat','vector.mlon','vector.kvect', $
+           'vector.mlat','vector.mlon','vector.kvect','vector.srng', $
            'vector.stid','vector.channel','vector.index', $
            'vector.vel.median','vector.vel.sd', $
            'vector.pwr.median','vector.pwr.sd', $
            'vector.wdt.median','vector.wdt.sd']
 
-  arrtype=[2,2,2,4,2,2,2,4,4,2,4,4,4,4,4,4,4,4,4,4,4,2,2,3,4,4,4,4,4,4]
+  arrtype=[2,2,2,4,2,2,2,4,4,2,4,4,4,4,4,4,4,4,4,4,4,4,2,2,3,4,4,4,4,4,4]
```

Comment:
Also recommend adding that comment here (appears to be the same type of array).

Reply:
Similiarly, I think we'll leave this one to resolve in #631